deployment-guide/upgrade-guides/upgrading-cinchy-versions/kubernetes-upgrades/v5.7-kubernetes.md

```diff
@@ -0,0 +1,203 @@
+---
+
+description: >
+This page details the instructions for upgrading your Cinchy platform to v5.7
+on Kubernetes
+
+---
+
+# v5.7 (Kubernetes)
+
+## What's new
```

Comment:
This was missing from previous upgrade guides. I think it's helpful to know the specific changes.

Reply:
Definitely it helps.